deployer/src/config_creator.py

```diff
@@ -68,6 +68,37 @@ def to_docusaurus_config(config, urls=None):
     return config
 
 
+def to_docusaurus_v2_config(config, urls=None):
+    if urls:
+        config["sitemap_urls"] = [
+            extract_root_from_input(urls[0]) + "sitemap.xml"]
+        config["sitemap_alternate_links"] = True
+    start_url = urls[0]
+    if '/docs/' not in start_url:
+        if not start_url.endswith('/'):
+            start_url = start_url + '/'
+
+        config["start_urls"] = [start_url + 'docs/']
+    else:
+        config["start_urls"] = [start_url]
```

Comment:
As you are already modifying `start_url` (line 79), continue doing it. Append 'docs/' to it in the if's true branch, and remove the else: it will make the data flow easier to read (as `config["start_urls"]` will get modified in only one place)

```python
if '/docs/' not in start_url:
  if not start_url.endswith('/'):
    start_url += '/'
  start_url += 'docs/'
config["start_urls"] = [start_url]
```